client-app/shared/catalog/components/product/variations.vue

```diff
@@ -56,7 +57,10 @@ interface IProps {
 defineProps<IProps>();
 
 function getProperties(variation: VariationType) {
-  return Object.values(getPropertiesGroupedByName(variation.properties ?? [], PropertyType.Product)).slice(0, 3);
+  const sortedByName = sortBy(variation.properties, (item) => item.name);
+  const variationProperties = sortBy(sortedByName, (item) => item.displayOrder);
```

Comment:
If I'm not mistaken - `sortBy` allows us to do it this way 
```js
sortBy(variation.properties, ['displayOrder', 'name'])
```

https://lodash.com/docs/4.17.15#sortBy